schema.sql

```diff
@@ -15,3 +15,23 @@ CREATE TABLE animals (
 );
   ALTER TABLE animals
 ADD COLUMN species VARCHAR;
+
+/* Create owners table */
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(100),
+    age INT
+);
+
+/* Create species table */
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100)
+);
+
+/* Remove species column */
+ALTER TABLE animals DROP COLUMN species;
+
+/* Add species_id in animals table referenced from species table */
+ALTER TABLE animals
+ADD COLUMN species_id INT REFERENCES species (id);
```

Comment:
- [ ] Please modify the `animals` table to add column `owner_id` which is a foreign key referencing the `owners` table

Reply:
Thank you, I did that